lib/galaxy/config/__init__.py

```diff
@@ -1200,6 +1201,9 @@ def _load_theme(path: str, theme_dict: dict):
         else:
             _load_theme(self.themes_config_file, self.themes)
 
+        if self.file_source_temp_dir:
+            self.file_source_temp_dir = os.path.abspath(self.file_source_temp_dir)
```

Comment:
This doesn't seem right to me, the check should be for `is None` as this is optional.

Reply:
This was based on the `tus_upload_store` config parameter, which also uses `new_file_path` as the default.

1. https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/schemas/config_schema.yml#L1963-L1969
2. https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L1002-L1003

Reply:
Ahh I see, I was expecting here to set the `new_file_path` as a fallback if the value was not set instead of doing the fallback at every usage. But it is fine then, there is only one usage anyway :+1: